scripts/docker/README.md

```diff
@@ -1,34 +1,56 @@
-# Trying out Elektra with Docker
+# Elektra Docker Artifacts
+This folder contains all Docker related Artifacts.
 
-This Dockerfile builds a dockerimage based on ubuntu:artful including all requirements to build Elektra.
+**Note:**
+Any commands in this file are expected to be run from the root
+of the repository.
 
-If [Docker](https://www.docker.com/) is available on your machine change to the directory containing the file you are currently reading and build the image with
-```sh
-$ docker build -t buildelektra .
-```
+## Building Images locally
+If you want to run or test Elektra via our Docker images you currently have
+to build them yourself.
+You can do so by running the following command:
 
-After the build process has completed you can create and run a Docker container that uses the image we just created.
-This runs bash in a container based on the buildelektra image and mounts the `./data` folder into the container.
 ```sh
-$ mkdir data
-$ docker run --rm -v ${PWD}/data:/mnt/share -it buildelektra bash
+docker build -t buildelektra-stretch-full \
+    --build-arg JENKINS_USERID=`id -u` \
+    --build-arg JENKINS_GROUPID=`id -g` \
+    -f scripts/docker/debian/stretch/Dockerfile \
+    scripts/docker/debian/stretch/
 ```
 
-The container contains a script that pulls a specified snapshot of Elektra from Github and either installs it in the container or builds a .deb package from it.
+You can adapt the targetted Dockerfile via `-f`.
+You should also adjust the tag used via `-t` if you are building a different
+image.
 
-Try it out from within the container with
-```sh
-# change into the mounted folder
-$ cd /mnt/share
-# this builds the commit with the tag "0.8.19"
-$ buildelektra "0.8.19"
-```
+Please note that the complete images used to test Elektra are quite big
+(~3.7GB) and take a some time (10min+) to build.
```

Comment:
Is it really 10min? On my fast machine with excellent Internet connection (TU network) it seemed to be longer.s

Reply:
Image creation is usually not capped by network speed but disk IO. On my local VM I can build the stretch image we use in 10 minutes. On the build servers it is a lot slower (especially on a7). It is even slower than expected honestly.